mmcv/runner/base_module.py

```diff
@@ -26,6 +43,14 @@ def __init__(self, init_cfg=None):
         self._is_init = False
         self.init_cfg = init_cfg
```

Comment:
- It is safer to deepcopy `init_cfg` when it is a dict.
- We need to specify the difference of `BaseModule` and `nn.Module` in the class docstring.
- Arguments should be listed in the class docstring.